{"pagelen": 100, "values": [{"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56443468.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56443468"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2020-03-07T15:00:55.259113+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56443468}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56443475.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56443475"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2020-03-07T15:01:49.959266+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56443475}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56443480.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56443480"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2020-03-07T15:02:41.759320+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56443480}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56443481.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56443481"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2020-03-07T15:03:00.241999+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56443481}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56443496.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56443496"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": null, "markup": "markdown", "html": "", "type": "rendered"}, "created_on": "2020-03-07T15:07:59.350536+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56443496}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56443511.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56443511"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": "If you check the environment used by Stephen Just to run Gazebo with Ogre 1.12 linked in [https://github.com/microsoft/vcpkg/pull/8178#issuecomment-552752416](https://github.com/microsoft/vcpkg/pull/8178#issuecomment-552752416), you can notice that he added `%gz_root_path%Media` to `GAZEBO_RESOURCE_PATH`. It may be a coincidence, but apparently the `ShadowVolume` folder that is referenced in the error, is indeed installed in `<prefix>/Media` by Ogre 1.12 \\(see [https://github.com/OGRECave/ogre/blob/v1.12.0/CMakeLists.txt#L501](https://github.com/OGRECave/ogre/blob/v1.12.0/CMakeLists.txt#L501)\\) and it was not installed by Ogre 1.11 .   \nPerhaps it may be worth a shot to try to add that directory to `GAZEBO_RESOURCE_PATH` and see if that solves the problem.\n\n\u200c", "markup": "markdown", "html": "<p>If you check the environment used by Stephen Just to run Gazebo with Ogre 1.12 linked in <a data-is-external-link=\"true\" href=\"https://github.com/microsoft/vcpkg/pull/8178#issuecomment-552752416\" rel=\"nofollow\">https://github.com/microsoft/vcpkg/pull/8178#issuecomment-552752416</a>, you can notice that he added <code>%gz_root_path%Media</code> to <code>GAZEBO_RESOURCE_PATH</code>. It may be a coincidence, but apparently the <code>ShadowVolume</code> folder that is referenced in the error, is indeed installed in <code>&lt;prefix&gt;/Media</code> by Ogre 1.12 (see <a data-is-external-link=\"true\" href=\"https://github.com/OGRECave/ogre/blob/v1.12.0/CMakeLists.txt#L501\" rel=\"nofollow\">https://github.com/OGRECave/ogre/blob/v1.12.0/CMakeLists.txt#L501</a>) and it was not installed by Ogre 1.11 . <br />\nPerhaps it may be worth a shot to try to add that directory to <code>GAZEBO_RESOURCE_PATH</code> and see if that solves the problem.</p>\n<p>\u200c</p>", "type": "rendered"}, "created_on": "2020-03-07T15:12:07.213974+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": "2020-03-07T15:12:50.721104+00:00", "type": "issue_comment", "id": 56443511}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56543816.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56543816"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": "I managed to run Gazebo built against Ogre 1.12 provided by vcpkg \\(in particular vcpkg version `2020.01` \\) by adding the `<ogre_install_prefix>/Media`  to `GAZEBO_RESOURCE_PATH` in the `setup.bat` . I guess a similar solution \\(or workaround\\) probably should work also with Ogre 1.12 provided by `conda-forge` .", "markup": "markdown", "html": "<p>I managed to run Gazebo built against Ogre 1.12 provided by vcpkg (in particular vcpkg version <code>2020.01</code> ) by adding the <code>&lt;ogre_install_prefix&gt;/Media</code>  to <code>GAZEBO_RESOURCE_PATH</code> in the <code>setup.bat</code> . I guess a similar solution (or workaround) probably should work also with Ogre 1.12 provided by <code>conda-forge</code> .</p>", "type": "rendered"}, "created_on": "2020-03-16T21:32:31.957520+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56543816}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56560106.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56560106"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": "Another \\(unrelated\\) problem that was present  with Ogre 1.12 is the one mentioned  \\(and fixed\\) in [#!/osrf/gazebo/pull-requests/3204/fix-ogre-fontmanager-getbyname-use-in-ogre/diff](#!/osrf/gazebo/pull-requests/3204/fix-ogre-fontmanager-getbyname-use-in-ogre/diff) .", "markup": "markdown", "html": "<p>Another (unrelated) problem that was present  with Ogre 1.12 is the one mentioned  (and fixed) in <a data-is-external-link=\"true\" href=\"#!/osrf/gazebo/pull-requests/3204/fix-ogre-fontmanager-getbyname-use-in-ogre/diff\" rel=\"nofollow\">#!/osrf/gazebo/pull-requests/3204/fix-ogre-fontmanager-getbyname-use-in-ogre/diff</a> .</p>", "type": "rendered"}, "created_on": "2020-03-17T23:12:58.158997+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56560106}, {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700/comments/56574078.json"}, "html": {"href": "#!/osrf/gazebo/issues/2700#comment-56574078"}}, "issue": {"links": {"self": {"href": "data/repositories/osrf/gazebo/issues/2700.json"}}, "type": "issue", "id": 2700, "repository": {"links": {"self": {"href": "data/repositories/osrf/gazebo.json"}, "html": {"href": "#!/osrf/gazebo"}, "avatar": {"href": "data/bytebucket.org/ravatar/{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}ts=1694483"}}, "type": "repository", "name": "gazebo", "full_name": "osrf/gazebo", "uuid": "{51a0cd5d-8697-4eb1-8b08-e919ee881e1c}"}, "title": "Gazebo 11 does not run correctly with Ogre 1.12 from conda-forge"}, "content": {"raw": "This  change is  explicitly mentioned in the Ogre 1.12 Changelog \\([https://github.com/OGRECave/ogre/blob/master/Docs/1.12-Notes.md#stable-media-files](https://github.com/OGRECave/ogre/blob/master/Docs/1.12-Notes.md#stable-media-files)\\): \n\n> **ACTION REQUIRED** you will have to add the `Media/ShadowVolume` resource location to use the build-in algorithms.\n\n\u200c", "markup": "markdown", "html": "<p>This  change is  explicitly mentioned in the Ogre 1.12 Changelog (<a data-is-external-link=\"true\" href=\"https://github.com/OGRECave/ogre/blob/master/Docs/1.12-Notes.md#stable-media-files\" rel=\"nofollow\">https://github.com/OGRECave/ogre/blob/master/Docs/1.12-Notes.md#stable-media-files</a>): </p>\n<blockquote>\n<p><strong>ACTION REQUIRED</strong> you will have to add the <code>Media/ShadowVolume</code> resource location to use the build-in algorithms.</p>\n</blockquote>\n<p>\u200c</p>", "type": "rendered"}, "created_on": "2020-03-18T22:24:19.959024+00:00", "user": {"display_name": "Silvio Traversaro", "uuid": "{34f404cb-5642-4f27-a032-e04c7143d776}", "links": {"self": {"href": "https://api.bitbucket.org/2.0/users/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D"}, "html": {"href": "https://bitbucket.org/%7B34f404cb-5642-4f27-a032-e04c7143d776%7D/"}, "avatar": {"href": "https://avatar-management--avatars.us-west-2.prod.public.atl-paas.net/557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6/578caaa1-056c-45a2-8bf9-bee94c3bebef/128"}}, "nickname": "traversaro", "type": "user", "account_id": "557058:e1c3e6a0-9a64-4e5f-a668-feebc191c5b6"}, "updated_on": null, "type": "issue_comment", "id": 56574078}], "page": 1, "size": 9}